### ISSUE TITLE
ddt4all crashes when trying to load screens

### ISSUE THREAD

**Issue by neilnutt** [cedricp/ddt4all#931]
I am able to load ddt4all and complete a scan. ddt4all unexpectedly quits a short moment after I click through to see a screen (screen never shows).  It doesn't matter which ecu I try to access, they all get the same unexpected quit with no warning raised.

Screenshot showing last moment before crash
![image](https://github.com/cedricp/ddt4all/assets/7341783/f029c36f-4308-44c4-a7ca-fda9f3d4428d)

Log file
[elm_ddt.log](https://github.com/cedricp/ddt4all/files/13232192/elm_ddt.log)

**Comment by AndyIreland** [comment]
what happens if you give this version a try - choose 'all cars' it has a built in database 
[https://www.drive2.ru/b/498093336985338243/](url)

**Comment by neilnutt** [comment]
Yes. The copy on the Russian site works. Doesn't resolve the issue for the current release, but fixes it for me.

**Comment by Furtif** [comment]
🧐

**Comment by KarelSvo** [comment]
> I am able to load ddt4all and complete a scan. ddt4all unexpectedly quits a short moment after I click through to see a screen (screen never shows). 


https://github.com/cedricp/ddt4all/discussions/930#discussioncomment-7490140
